libcgt-config.cmake.in

```diff
@@ -0,0 +1,15 @@
+@PACKAGE_INIT@
+
+include( CMakeFindDependencyMacro )
+
+# List dependencies.
```

Comment:
I could not find what this notation means after 10 minutes googling. Can you have a short explanation or if its a shortcut for a command, use the full version? 

Actually, finally got it. I think a comment of how this system of dependencies would be helful :-)